src/components/autonomous/WaypointNav.js

```diff
@@ -18,6 +20,17 @@ function WaypointNav() {
     dispatch(requestWaypointNav(formJson));
   };
 
+  function grabFromClipboard () {
+    navigator.clipboard.readText().then(text => {
+      if(text.match("-?\\d{1,}\\.?\\d{0,}, -?\\d{1,}\\.?\\d{0,}")) {
```

Comment:
The quantifiers are a bit verbose, `{1,}` is equivalent to `+` and `{0,}` is equivalent to `*`, so this would be cleaner: `-?\\d+\\.?\\d*, -?\\d+\\.?\\d*`. Also, for the benefit of someone who isn't as well versed in regexes, it would be good to add a comment explaining what this regex matches.